README.md

```diff
@@ -104,6 +104,27 @@ Sometimes you'll want to add a comment which is only for documentation maintaine
 
 Use `{/* my comment */}` rather than the HTML-style comments you'd normally use for Markdown files. Other comment types will cause errors.
 
+### Task: Merging `master` into `release-next`
+
+In rare occasions, when writing documentation for an unreleased feature, you may
+notice that some recent changes in `master` aren't present in `release-next`. If
+that is a problem, you will want to update `release-next` branch with the latest
+changes from `master`. To update `release-next` with the changes made to
+`master`, follow these steps:
+
+1. Create a PR to merge `master` into `release-next` using [this magic
+   link][ff-release-next].
+2. If you see the label `dco-signoff: no`, add a comment on the PR with:
+
+   ```text
+    /override dco
+    ```
+
+   It is necessary because some the merge commits have been written by the bot
+   and do not have a DCO signoff.
```

Comment:
I wonder if it's possible to configure the bot to use --signoff when merging?

Reply:
I think @inteon investigated this idea ([message](https://github.com/jetstack/testing/pull/872#issuecomment-1506580069)):

> From what I found it is not possible to add DCO signoff in the commit message. However, I found that the DCO check should actually check if the committer is a bot and not complain in that case, this is not well-documented and needs further investigation. For now, I found that just merging in case all other checks are OK works fine.

Reply:
That was specific for that generic autobumper tool.
I don't know what tool is used here.